dspace/config/crosswalks/oai/metadataFormats/qdc.xsl

```diff
@@ -26,6 +26,16 @@
 			<xsl:for-each select="doc:metadata/doc:element[@name='dc']/doc:element[@name='title']/doc:element/doc:field[@name='value']">
 				<dc:title><xsl:value-of select="." /></dc:title>
 			</xsl:for-each>
+			<xsl:for-each select="doc:metadata/doc:element[@name='dc']/doc:element[@name='creator']/doc:element/doc:field[@name='value']">
+				<dc:creator>
+					<xsl:value-of select="." />
+				</dc:creator>
+			</xsl:for-each>
+			<xsl:for-each select="doc:metadata/doc:element[@name='dc']/doc:element[@name='contributor']/doc:element[@name='author']/doc:element/doc:field[@name='value']">
```

Comment:
This change does more than add dc.creator, it also adds dc.contributor.author... which is probably a necessary change/bugfix, but has not yet been mentioned as an issue.